lib/mpdecimal.ex

```diff
@@ -1,12 +1,26 @@
 defmodule MPDecimal do
   alias MPDecimal.Nif
 
+  @doc """
+  raise a decimal to a power
+
+  This function performs exponentiation with fixed-decimal precision.
+
+  ## Examples
+
+      iex> two = Decimal.new("2.0")
```

Comment:
these examples turn into unit tests thanks to the [DocTest](https://elixir-lang.org/getting-started/mix-otp/docs-tests-and-with.html) that we have setup in `test/mpdecimal_test.exs`